rdr_service/resource/tasks.py

```diff
@@ -93,3 +93,21 @@ def batch_rebuild_retention_metrics_task(payload):
         count += 1
 
     logging.info(f'End time: {datetime.utcnow()}, rebuilt {count} Retention Metrics records.')
+
+def batch_rebuild_consent_metrics_task(payload):
+    """
+     Rebuild a batch of consent metrics records based on ids from the consent_file table
+     :param payload: Dict object with list of ids to work on.
+     """
+    res_gen = generators.ConsentMetricsGenerator()
+    batch = payload.get('batch')
+
+    count = 0
+    # Retrieve the consent_file table records by id
+    results = res_gen.get_consent_validation_records(id_list=batch)
+    for row in results:
```

Comment:
Probably better to use enumeration in all these for-loop instances instead of having to use an incremental operator since there is no conditional logic used for the increment.

Reply:
Done